### ISSUE TITLE
Merge/Link Tracks for additional FX per track

### ISSUE THREAD

**Issue by michaalhell** [polyend/TrackerIssues#1127]
While 8 tracks are great, i find myself not using them all at times. 

For me the ability to be able to merge one track with another so that you get additional FX slots would be an incredible enhancement. my main pet peeve with the tracker is that i want more FX goodness per track and i feel like this would be a great way of doing so without sacrificing the great workflow that the tracker has. 

so basically the option to merge tracks on the pattern screen. another way could be to add an option in the config section, basically pre determening the amount of tracks you have. so if you want 6 tracks you get two tracks with 4 FX slots and 4 tracks with 2 fx slots, etc. and maybe be so bold as to merge all the tracks for one track with 16 fx slots :D. 

anyways please consider
